cst438-week2/.gitignore

```diff
@@ -6,9 +6,7 @@ target/
 
 ### STS ###
 .apt_generated
-.classpath
```

Comment:
This change was needed for Github interoperability with Eclipse.

Reply:
Good change.